### ISSUE TITLE
Wrong annotations in order model

### ISSUE THREAD

**Issue by lopes-vincent** [thelia/thelia#1728]
In Model/Order.php all choosenAddress (delivery and invoice) getters and setters have wrong annotations, for example :

```
    /**
     * @return Address
     */
    public function getChoosenDeliveryAddress()
    {
        return $this->choosenDeliveryAddress;
    }
```

the type is set to address but in reality choosenDeliveryAddress is address id not the object.

**Comment by manuelbcd** [comment]
Solved. Waiting to pull request revision. Regards

**Comment by manuelbcd** [comment]
Sorry, it was solved previously.  https://github.com/thelia/thelia/pull/1701

**Comment by bibich** [comment]
Fixed by #1701
